<ui version="4.0" >
 <class>General_tab_ui</class>
 <widget class="QWidget" name="General_tab_ui" >
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <property name="sizeConstraint" >
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="action_name_lineedit" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="action_type_label" >
       <property name="text" >
        <string>Action &amp;type:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>action_type_combo</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="action_type_combo" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>&amp;Disabled</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>disable_checkbox</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QCheckBox" name="disable_checkbox" />
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="comment_label" >
       <property name="text" >
        <string>&amp;Comment:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>comment_multilineedit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3" >
      <widget class="KTextEdit" name="comment_multilineedit" />
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="action_name_label" >
       <property name="text" >
        <string>Action &amp;name:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>action_name_lineedit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="global" >kdialog.h</include>
  <include location="local" >ktextedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
